app/webpacker/components/Tickets/index.jsx

```diff
@@ -51,12 +53,25 @@ function TicketContent({ ticketDetails, currentStakeholder, sync }) {
         sync={sync}
         currentStakeholder={currentStakeholder}
       />
+      <TicketComments
+        ticketId={ticketDetails.ticket.id}
+        currentStakeholder={currentStakeholder}
+      />
       <TicketLogs logs={ticketDetails.ticket.ticket_logs} />
```

Comment:
In `TicketLogs`, you already hard-pass the logs. In `TicketComments`, you only pass the ID and fetch the comments dynamically. Both approaches are fine by themselves, but why do you mix/combine them in this unusual way?